File: templates/Default.py
def solution():
    # Solution {N}


try:
    import sys
    import os
    if (os.path.exists(".cpl_files/{N}.in.txt")):
        sys.stdin = open(".cpl_files/{N}.in.txt","r")
        sys.stdout = open(".cpl_files/{N}.out.txt","w")
except:
    pass

t = int(input())
while t:
    solution()
    t -= 1

File: templates/Default.java
import java.util.*;
import java.lang.*;
import java.io.*;


public class Main {
    static Scanner sc;


    static void solution() throws Exception {
        // Solution {N}
    }


    public static void main(String[] args) throws Exception {
        InputStream is = System.in;
        PrintStream ps = System.out;
        try {
            if (System.getProperty("ONLINE_JUDGE") == null) {
                is = new FileInputStream(".cpl_files/{N}.in.txt");
                ps = new PrintStream(".cpl_files/{N}.out.txt");
            }
        } catch (Exception e) {}
        sc = new Scanner(is);
        System.setOut(ps);
        int t;
        t = sc.nextInt();
        while (t-- > 0) {
            solution();
        }
        is.close();
        ps.close();
    }
}
